server/data/userClient.ts

```diff
@@ -28,6 +28,7 @@ export default class UserClient {
   }
 
   async getUsers(
+    region?: string,
```

Comment:
I think it would make sense to follow the pattern of the other params in this function and give `region` a default value